src/SFA.DAS.LevyTransferMatching.Web/Orchestrators/PledgeOrchestrator.cs

```diff
@@ -220,6 +224,60 @@ public async Task<LocationSelectViewModel> GetLocationSelectViewModel(LocationSe
             };
         }
 
+        public async Task<byte[]> GetPledgeApplicationsDownloadModel(ApplicationsRequest request)
+        {
+            var result = await _pledgeService.GetApplicationsForCsvDownload(request.AccountId, request.PledgeId);
+
+            var pledgeAppModel = new PledgeApplicationsDownloadModel
+            {
+                Applications = result.Applications?.Select(app =>
+                {
+                    var appStandard = app.Standard;
+                    var affordability = GetAffordabilityViewModel(app.Amount, app.PledgeRemainingAmount,
+                        app.NumberOfApprentices, app.MaxFunding, app.EstimatedDurationMonths, app.StartBy);
+
+                    return new PledgeApplicationDownloadModel
+                    {
+                        DateApplied = app.DateApplied,
+                        Status = app.Status,
+                        ApplicationId = app.ApplicationId,
+                        PledgeId = app.PledgeId,
+                        Locations = app.Locations ?? new List<string>(),
+                        EmployerAccountName = app.EmployerAccountName,
+                        HasTrainingProvider = app.HasTrainingProvider,
+                        Sectors = app.Sectors ?? new List<string>(),
+                        AboutOpportunity = app.AboutOpportunity,
+                        BusinessWebsite = GetUrlWithPrefix(app.BusinessWebsite),
+                        FormattedEmailAddress = String.Join(";", app.EmailAddresses),
+                        FormattedSectors = String.Join(",", app.Sectors ?? new List<string>()),
+                        FirstName = app.FirstName,
+                        LastName = app.LastName,
+                        NumberOfApprentices = app.NumberOfApprentices,
+                        StartBy = app.StartBy,
+                        TypeOfJobRole = app.JobRole,
+                        EncodedPledgeId = _encodingService.Encode(app.PledgeId, EncodingType.PledgeId),
+                        EncodedApplicationId =
+                            _encodingService.Encode(app.ApplicationId, EncodingType.PledgeApplicationId),
+                        FormattedLocations = String.Join(",", app.Locations ?? new List<string>()),
+                        IsJobRoleMatch = app.IsJobRoleMatch,
+                        IsLevelMatch = app.IsLevelMatch,
+                        IsLocationMatch = app.IsLocationMatch,
+                        IsSectorMatch = app.IsSectorMatch,
+                        Duration = appStandard.ApprenticeshipFunding.GetEffectiveFundingLine(app.StartBy).Duration,
```

Comment:
Duration can be exposed from inner/outer api now, and not be calculated from the funding lines